### ISSUE TITLE
Detect missing @expand

### ISSUE THREAD

**Issue by jayvdb** [zuo/unittest_expander#3]
If `@expand` is omitted, the result on py37 is

```
                with outcome.testPartExecutor(self, isTest=True):
>                   testMethod()
E                   TypeError: test_package() missing 1 required positional argument: 'name'
```

There should be a way to report a more user-friendly /informative error message before this `TypeError` occurs.

**Comment by zuo** [comment]
Yes, `@foreach` should wrap the decorated method with a callable that raises `TypeError` with a more informative message (and `@expand` would do the corresponding unwrapping).

For sure, it would be a usability improvement.

Thank you. :-)

**Comment by zuo** [comment]
Hello.

@jayvdb Big thanks for you contribution!

I apologise for such a long inactivity -- for personal reasons I was unable to actively maintain the project, but now that will change. I hope I'll be able to answer to your ideas in the coming weeks.

Best regards!

**Comment by zuo** [comment]
Hmmm, I am still not sure how to provide the feature in a possibly non-invasive way, which will not prevent me from adding some other features... I will revisit this issue when come up with a satisfying answer.
